src/plugins/data_view_field_editor/public/service/format_editor_service.ts

```diff
@@ -52,7 +52,7 @@ export class FormatEditorService {
       truncateFormatEditorFactory,
       urlFormatEditorFactory,
       histogramFormatEditorFactory,
-    ];
+    ] as FieldFormatEditorFactory[];
 
```

Comment:
I attempted and failed to revert this change. There's something I'm not understanding about default types and type params.